virttest/utils_test/libvirt.py

```diff
@@ -2580,7 +2580,9 @@ def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
         return False
 
 
-def update_vm_disk_source(vm_name, disk_source_path, source_type="file"):
+def update_vm_disk_source(vm_name, disk_source_path,
+                          disk_image_name="",
+                          source_type="file"):
```

Comment:
Where's the caller of update_vm_disk_source?
Due to you add new param, It's better to check if it will affect all of the callers.